src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -334,7 +334,7 @@ PARAM_DEFINE_FLOAT(MPC_TILTMAX_AIR, 45.0f);
  *
  * @unit deg
  * @min 0.0
- * @max 90.0
+ * @max 170.0
```

Comment:
feeling adventurous? 😄 It's a good idea to check corner angles (first in simulation!) to verify the implementation. 180 should theoretically also work or do we run into gimbal lock problems with the setpoint generation?

Reply:
Yes, see the commit message: https://github.com/PX4/Firmware/pull/10805/commits/78da9ff79225f8fce638821e5a359e85b531990c. 180 is fine setpoint-wise, beyond that the yaw inverts. I wanted to account for overshoots, but I realize now that even with overshoots the attitude should be tracked correctly up to 180 degrees setpoints.

Reply:
I tested it again and 180 works fine. Interestingly even above that works now. I increased the limit to 180.